src/Sarif.Viewer.VisualStudio/Themes/DefaultStyles.xaml

```diff
@@ -49,6 +49,23 @@
                 Value="Courier New" />
         <Setter Property="FontSize"
                 Value="12" />
+        <Style.Triggers>
+            <DataTrigger Binding="{Binding Location.Importance}"
+                         Value="Essential">
+                <Setter Property="FontWeight"
+                        Value="Bold" />
```

Comment:
... and for that matter you could get rid of the `ImportanceToForegroundConverter` and its unit tests, and use style triggers for that as well.